### DISCUSSION TITLE
gtwr model

### DISCUSSION THREAD

**Discussion by yangcheng258** [pysal/pysal#1359] [Q&A]
Dear team, is there any demo for the geographically and temporally weighted regression?

**Comment by TaylorOshan** [comment]
Greetings,

This is not currently supported within any of the current packages. There’s
just GWR and MGWR, but no temporal extensions available at the moment.

Best,
Taylor

**Reply by yangcheng258** [reply]
Thank you so much! Looking forward to this option the future!
